lib/src/scss-shared/_nx-colors.scss

```diff
@@ -131,3 +76,66 @@ $nx-selectable-colors: (
   yellow: ($nx-yellow-700, $nx-yellow-50),
   lime: ($nx-green-200, $nx-green-50)
 );
+
+/*
+  WARNING
```

Comment:
I think this section would be a lot better as a separate file than a "section" at the bottom of this file